### ISSUE TITLE
[TC-TSTAT-2.1 & 2.2] YAML script needs to be updated with adding missing steps as per the test plan

### ISSUE THREAD

**Issue by KishokG** [project-chip/matter-test-scripts#451]
## Description:

- In TC-TSTAT-2.1: Test step 50 to 60 needs to be added and step 25(AlarmMask) needs to be removed as per the test plan.
- In TC-TSTAT-2.2: Test step 19 to 22c needs to be added in the YAML script as per the test plan.

**Test Plan Reference:**  https://github.com/CHIP-Specifications/chip-test-plans/blob/master/src/cluster/thermostat.adoc#tc-tstat-2-1-attributes-with-server-as-dut

**Comment by nivi-apple** [comment]
AlarmMask (25) was removed as part of https://github.com/CHIP-Specifications/chip-test-plans/commit/08823fb42e64cc48fe948ec4f15d66d117b79ae0

**Comment by KishokG** [comment]
Yes @nivi-apple it's removed from the test plan, the same needs to be removed from the script as well.

**Comment by nivi-apple** [comment]
@KishokG My bad for the incorrect link. the test scripts don't seem to have alarm mask in any steps.

**Comment by nivi-apple** [comment]
Fixes: https://github.com/CHIP-Specifications/chip-test-plans/pull/4855 and https://github.com/project-chip/connectedhomeip/pull/37052. If we agree with the fix the test plan needs to merge first and then the test scripts since we removed test section 4.1 and moved those into 2.1
